sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationConfig.kt

```diff
@@ -0,0 +1,13 @@
+package com.mapbox.maps.plugin.annotation
+
+/**
+ * Config class for annotation plugin to get the annotation manager
+ */
+data class AnnotationConfig(
+  /** The id of the layer above the annotation layer*/
+  val belowLayerId: String? = null,
```

Comment:
The doc states that `id of the layer above the annotation layer` while val is named `belowLayerId` 🤔

Reply:
We will add the annotation layer below the given layer, which is the same as the definition in Layer